samples/tab-add-in-combined/nodejs/README.md

```diff
@@ -0,0 +1,181 @@
+---
+page_type: sample
+description: Sample Outlook add-in and Teams tab app.
+products:
+- office-teams
+- office
+- office-365
+languages:
+- typescript
+- nodejs
+extensions:
+ contentType: samples
+ createdDate: "05/15/2023 01:00:00 PM"
+urlFragment: officedev-microsoft-teams-samples-multiple-capability-discounts-ts
+---
+
+# Discount Offers
+
+This sample combines, into a single app with a single manifest, an Outlook Add-in with a Teams Tab to make and manage discounts offers. The Outlook add-in is used to insert discount offers into e-mails. These offers are recorded in a mock customer relation management (CRM) system that is implemented with a simple JSON database. The aggregated data from the CRM is viewed in a Teams tab.
+
+The sample illustrates a prime use case for combining a Teams Tab and an Office Add-in: Users take discrete actions on an Office document and analyze data about these discrete actions in a Teams tab.
+
+**Note:** As of 6/20/2023, Teams apps that include an Office Add-in capability are in preview and can run only on a Windows computer. You should be working on a Windows computer to run this sample.
+
+## Included Features
+
+- Tabs
+- Office Add-ins
+
+## Interaction with app
+
+Interaction with the add-in:
+
+   ![Animated gif shows a discount offer being inserted into an email with the Discounts add-in](Images/interact-add-in.animated.gif)
+
+Interaction with the tab:
+
+   ![Animated gif shows a discount offer being inserted into an email with the Discounts add-in](Images/interact-tab.animated.gif)
+
+## Prerequisites
+
+- [NodeJS](https://nodejs.org/en/), version 16.x.x or 18.x.x.
+- [npm](https://www.npmjs.com/) (installed with NodeJS)
+
+    **Note:** This sample was developed with NodeJS version 16.13.2 and npm version 8.1.2. It should work with most NodeJS versions in the 16.x.x or 18.x.x ranges, but there are some exceptions to this. It does not work on NodeJS 16.16.0.
+
+- [Teams](https://teams.microsoft.com) Microsoft Teams is installed and you have an account.
+- [Visual Studio Code](https://code.visualstudio.com/)
+- [Teams Toolkit extension for Visual Studio Code](https://learn.microsoft.com/microsoftteams/platform/toolkit/install-teams-toolkit?tabs=vscode)
+- [M365 developer account](https://docs.microsoft.com/microsoftteams/platform/concepts/build-and-test/prepare-your-o365-tenant) or access to a Teams account with the appropriate permissions to install an app.
+- Microsoft Office for Windows
+
+    **Note:** As of 6/20/2023, the add-in capability in a Teams app is only supported on Windows computers.
```

Comment:
Pease update it if possible.

Reply:
Will cover in different new PR

Reply:
https://github.com/OfficeDev/Microsoft-Teams-Samples/pull/965